docs/_redirects

```diff
@@ -1,3 +1,44 @@
+# 2024 SEO audit
+/nodes/nodes.html    /integrations/builtin/node-types/
+/getting-started/installation/advanced/server-setup.html    /hosting/installation/server-setups/
+/getting-started/installation/advanced/server-setup.html#server-setups  /hosting/installation/server-setups/
+/getting-started/key-components.html    /workflows/
+/integrations/buil%20%20%20%20%20%20%20%20%20%20%205tin/core-nodes/n8n-nodes-base.start/    /integrations/builtin/core-nodes/n8n-nodes-base.start/
+/getting-started/create-your-first-workflow/*    /try-it-out/
+/reference/security/   /privacy-security/security/
+/reference/reference.html   /
+/integrations/builtin/core-nodes/n8n-nodes-base.executionData  /integrations/builtin/core-nodes/n8n-nodes-base.executiondata/
+
+
+# Moved from 404.html
+/getting-started/key-concepts/   /flow-logic/
+/getting-started/key-concepts/#expression   /code/expressions/
+/getting-started/key-concepts/#item   /data/data-structure/
+/getting-started/key-concepts/#function   /data/code/
+/getting-started/key-concepts/#data   /data/
+/getting-started/key-concepts/#data-structure   /data/data-structure/
+/key-concepts/#data-flow   /data/data-structure/
+/key-concepts/#error-workflow   /flow-logic/error-handling/
+/key-concepts/#security   /hosting/configuration/user-management-self-hosted/
+/getting-started/whats-next.html    /try-it-out/quickstart/#next-steps
+/getting-started/installation/advanced/server-setup.html#docker-compose-examples    /hosting/installation/server-setups/docker-compose/
+/reference/faq.html#Integrations    /integrations/
+/reference/faq.html     /faircode-license/
+/reference/faq.html#License     /faircode-license/
+/reference/faq.html#n8n-cloud   /manage-cloud/overview/
+/nodes/expressions.html#expressions     /code/expressions/
+/editor-ui/workflows    /workflows/
+/editor-ui/credentials     /credentials/
+/editor-ui/executions   /workflows/executions/
+/editor-ui/admin-panel  /manage-cloud/update-cloud-version/
+/#what-is-n8n   /
+/getting-started/tutorials.html#blogposts   https://n8n.io/blog/tag/tutorial/
+/#server-setup     /hosting/installation/server-setups/
```

Comment:
@StarfallProjects this redirect came from 404.html. I edited the rule slightly. It used to list the old URL as /#/server-setup, and that redirect did not work if you hit the old URL. I assumed that it was supposed to be an anchor on the page, so I've removed the second "/", but wanted to call your attention to this one since it was already a bit funky.